src/daemon/ca/commands.rs

```diff
@@ -348,6 +358,14 @@ impl CmdDet {
         eventsourcing::SentCommand::new(handle, None, CmdDet::ChildRemove(child_handle), actor)
     }
 
+    pub fn child_suspend_inactive(handle: &Handle, child_handle: ChildHandle, actor: &Actor) -> Cmd {
```

Comment:
Same as above, is there really any notion of inactiveness here or is this just a command to suspend and the caller determines whether it is suspending due to inactivity or for some other reason?